lib/funneler.rb

```diff
@@ -16,7 +16,10 @@ class << self
     attr_reader :configuration
 
     def configuration
-      @configuration ||= Funneler::Configuration.new
+      @configuration ||= Funneler::Configuration.new(
+        jwt_key: ENV.fetch("JWT_KEY"),
```

Comment:
I don't think we should do this here, we'll be better off explicitly configuring it using an initializer like:
```
Funneler.configure do |config|
  config.jwt_key = "foobar"
  config.jwt_algorithm = "HS256"
end
```